scripts/lint/check-jsp-size.sh

```
@@ -0,0 +1,99 @@
+    line="${line%%*([[:space:]])}"
+    [[ -z "$line" ]] && continue
+    ALLOWLIST["$line"]=1
+  done < "$ALLOWLIST_FILE"
+fi
+
+fails=0
+allowlist_hits=0
+
+# Find all JSPs under billing/
+while IFS= read -r -d '' jsp; do
+  rel="${jsp#$REPO_ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)